### PR TITLE
Feature covidscore dan

### PR DESCRIPTION
**Pull request by danoand**
Creating a draft PR for discussion purposes.  Comments welcome but let's hold off on merging for now

### CODE REVIEW

File: project/app/api/covid_score.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        "error":    error message (if applicable)  
+    """
+    # Define a return object
+    ret_dict = {
+        "ok": False,
+        "new_case":   None,
+        "data":       None,
+        "error":      "an error occurred"
+    }
+
```
Comment (unburied) [inline project/app/api/covid_score.py:85]: Nice documentation here

File: project/app/api/covid_score.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    Parameters:
+        "state": the targeted state (e.g. "CA")
+        "date":  the date of the covid data requested
+
+    Returns:
+        "ok":       boolean indicating a successful request
+        "new_case": new cases logged in the API response
+        "data":     copy of the CDC API response
+        "error":    error message (if applicable)  
+    """
```
Comment (unburied) [inline project/app/api/covid_score.py]: Nice documentation here

File: project/app/api/covid_score.py [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    # Define a return object
+    ret_dict = {
+        "ok": False,
+        "new_case":   None,
+        "data":       None,
+        "error":      "an error occurred"
```
Comment (unburied) [inline project/app/api/covid_score.py:87]: Maybe error should be none too? If no error found, nothing to do.

**Review by unburied — COMMENTED** [review]
Looks good

File: project/app/api/covid_score.py (hunk 3 above)
Comment (AmyBeisel) [reply project/app/api/covid_score.py:87]: https://www.latimes.com/science/story/2020-11-11/covid-19-risk-thanksgiving-2020-what-to-expect - or something similar to this "No or missing data"?

**Comment by danoand** [comment]
Closing this PR and will open a new PR to reflect changes to support grabbing covid stats from the database (rather than pulling from a third party API in real time)

Closing...

**Pull request closed without merging**